File: JAVA_SIS104/app/src/main/java/com/example/prueba_java/backend/operaciones.java
package com.example.prueba_java.backend;

public class operaciones {
    private int x;
    private int y;

    public operaciones(int x, int y) {
        this.x = x;
        this.y = y;
    }

    public int getX() {
        return x;
    }

    public int getY() {
        return y;
    }

    public void setX(int x) {
        this.x = x;
    }

    public void setY(int y) {
        this.y = y;
    }

    public int suma(){
        return this.x+this.y;
    }

    public int resta(){
        return this.x-this.y;
    }

    public int multiplicacion(){
        return this.x*this.y;
    }

    public int division(){
        return this.x/this.y;
    }
}

File: KOTLIN_SIS104/app/src/main/java/com/example/prueba_kotlin/backend/operaciones.kt
package com.example.prueba_kotlin.backend

class operaciones(private var x: Double, private var y: Double){
    fun suma(): Double{
        return this.x + this.y
    }

    fun resta(): Double{
        return this.x - (this.y)
    }

    fun multiplicacion(): Double{
        return this.x * this.y
    }

    fun division(): String{
        if(this.y != 0.0){
            return (this.x/this.y).toString()
        }else{
            return "No se puede dividir"
        }
    }

    fun setX(num1: Double){
        this.x =  num1
    }

    fun getX(): Double{
        return this.x
    }

    fun setY(num2: Double){
        this.y =  num2
    }

    fun getY(): Double{
        return this.y
    }
}